src/server/index.tsx

```diff
@@ -78,6 +78,10 @@ export const createServer = ({
   const staticSrc = env.PUBLIC_URL
     ? `${env.PUBLIC_URL}${STATIC_PATH}`
     : "'none'";
+  const imgSrc = [staticSrc];
+  if (env.PUBLIC_URL) {
```

Comment:
Why is `data:` only added when `PUBLIC_URL` exists? How are they related?

Reply:
Tbh, I'm not sure, but I was trying to preserve the logic from above which seems to say "if `env.PUBLIC_URL` is falsey, then set the value to `"'none'"`. But I could very well not understand what's going on here. Without this `if`, we end up with an `imgSrc` of `["'none'", 'data:']` when `env.PUBLIC_URL` is falsey. Is that what we want?

Reply:
Ok, I made that change. You can comment on re-review if this is what we want.